From: Michael Biebl <biebl@debian.org>
Date: Fri, 17 May 2019 08:02:46 +0200
Subject: meson: stop creating .wants directories for
 {multi-user,getty}.target (#12569)

Since preset is supposed to be used to enable the services, there is no
need to pre-create those directories either.

Follow-up for #12164

(cherry picked from commit dadc7f2e43b163dd740832abd73b83245f716c1e)
---
 src/core/meson.build | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/core/meson.build b/src/core/meson.build
index 86ab35d..f1df760 100644
--- a/src/core/meson.build
+++ b/src/core/meson.build
@@ -212,10 +212,6 @@ meson.add_install_script('sh', '-c', mkdir_p.format(systemsleepdir))
 meson.add_install_script('sh', '-c', mkdir_p.format(systemgeneratordir))
 meson.add_install_script('sh', '-c', mkdir_p.format(usergeneratordir))
 
-meson.add_install_script('sh', '-c',
-                         mkdir_p.format(join_paths(pkgsysconfdir, 'system/multi-user.target.wants')))
-meson.add_install_script('sh', '-c',
-                         mkdir_p.format(join_paths(pkgsysconfdir, 'system/getty.target.wants')))
 meson.add_install_script('sh', '-c',
                          mkdir_p.format(join_paths(pkgsysconfdir, 'user')))
 meson.add_install_script('sh', '-c',
